crates/re_types/tests/fuzzy.rs

```diff
@@ -277,7 +276,7 @@ fn roundtrip() {
     };
 
     let arch = AffixFuzzer1::new(
-        fuzzy1.clone(),
+        fuzzy1.clone(), //
```

Comment:
oopsie?

Reply:
On purpose, the ol' "leave me in peace rustfmt" trick